src/App.tsx

```diff
@@ -37,21 +36,17 @@ const useStyles = makeStyles((theme: Theme) =>
 export default function App() {
   const classes = useStyles();
 
-  const [measureFileName, setMeasureFileName] = useState<string | null>(null);
-  const [patientFileName, setPatientFileName] = useState<string | null>(null);
+  let [measureFileName, setMeasureFileName] = useState<string | null>(null);
+  let [patientFileName, setPatientFileName] = useState<string | null>(null);
 
-  const [loading, setLoading] = useState<boolean>(false);
-  const [results, setResults] = useState<any>(null);
-
-  const [measureBundle, setMeasureBundle] = useState<any>(null);
-  const [patientBundle, setPatientBundle] = useState<any>(null);
+  let [measureBundle, setMeasureBundle] = useState<any>(null);
+  let [patientBundle, setPatientBundle] = useState<any>(null);
```

Comment:
I'm guessing you changed these to `let`s because you were initially trying to change the value of the state variable itself directly rather than using the setter. These can be changed back to `const`s safely now I believe